_posts/2018-12-14-Bazel-0.20.0.md

```diff
@@ -62,13 +62,12 @@ Breaking changes in the next release (0.21) [are marked with GitHub label 'break
 
   - Added support for Android NDK r18.
   - Make legacy-test-support (`legacy_test-<api-level>`) from
-    `android_sdk_repository` neverlink. The legacy test support
-    libraries shouldn't be built into test binaries. To make them
-    available at runtime, developers should declare them via
-    uses-library:
-    https://developer.android.com/training/testing/set-up-project#android-test-base
+    `android_sdk_repository` neverlink. The legacy test support libraries
+    shouldn't be built into test binaries. To make them available at runtime,
+    [developers should declare them via
+    uses-library](https://developer.android.com/training/testing/set-up-project#android-test-base).
     
- ## Starlark
+## Starlark
```

Comment:
Do we need an empty line here?

Reply:
Don't think so, but I added it to be consistent anyway.